docs/guide/snaps-exports.md

```diff
@@ -134,3 +134,69 @@ module.exports.onTransaction = async ({
   return { insights };
 };
 ```
+
+## `onCronjob`
+
+If a snap wants to to run periodic actions for the user, the snap must export a function called `onCronjob`. This function will be called at the specified times with the specified payloads defined in the `endowment:cronjob` permission.
+
+::: tip Requesting the cronjob permission
+In order for the extension to call the `onCronjob` method of the snap, the `endowment:cronjob` permission must be requested. See [Permissions](./snaps-permissions.html#endowment-cronjob)
+:::
+
+### Parameters
+
+- `onCronjobArgs` - exclusively containing an RPC request specified in the `endowment:cronjob` permission.
+
+```typescript
+interface onCronjobArgs {
+  request: JsonRpcRequest<unknown[] | { [key: string]: unknown }>;
+}
+```
+
+### Examples
+
+#### Typescript
+
+```typescript
+import { OnCronjobHandler } from '@metamask/snap-types';
+
+export const onCronjob: OnCronjobHandler = async ({ request }) => {
+  switch (request.method) {
+    case 'hello':
+      return wallet.request({
+        method: 'snap_notify',
+        params: [
+          {
+            type: 'inApp',
+            message: `Hello, world!`,
+          },
+        ],
+      });
+
+    default:
+      throw new Error('Method not found.');
+  }
+};
+```
+
+#### Javascript
+
+```js
+module.exports.onCronjob = async ({ request }) => {
+  switch (request.method) {
+    case 'hello':
```

Comment:
Maybe matching the name of the method with the one defined in specification would be better.
`hello` -> `exampleMethodOne` or whatever you decide to have in a specification.

Reply:
Put a username?